src/Components/Products.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { addCartItemThunk, createOrderThunk, addGuestItem } from '../redux/store';
+
+class _Products extends Component {
+  create(e, product) {
+    const { auth, cart, orders, createOrder, addToCart, addGuestItem } = this.props;
+    e.preventDefault();
+
+    //console.log(this.props.cart, this.props.orders);
+
+    // Gets cart for specific user
+    if (auth.name !== 'Guest') {
```

Comment:
We keep seeing this guest specific logic, I don't have a firm answer about what is better, but I think something is.